presentation/src/main/res/layout/fragment_lets_meet_in_wroclaw.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/card_view_layout"
+    style="@style/default_card_style"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingBottom="@dimen/size_3x">
+
+        <View
```

Comment:
Previously we have created header component. `com.safetyheads.akademiaandroida.presentation.ui.components.H2View`
use it, instead of `View` and `TextView`

Reply:
I agree, but there is no green view on the left in H2View.

Reply:
Good idea, corrected